app/src/templates/image.html

```diff
@@ -3,6 +3,20 @@
 <div class="modal-header">
     <h3 class="modal-title"><span editable-text="ic.CourseCache.current_image.title" onbeforesave="ic.save()">{{ ic.CourseCache.current_image.title || "Click to add a Title" }}</span></h3>
     <pp><span editable-text="ic.CourseCache.current_image.description" onbeforesave="ic.save()">{{ ic.CourseCache.current_image.description || "Click to add a Description" }}</span></pp>
+    <ul class="list-unstyled">
+      <li ng-repeat="meta in ic.image.metadata track by $index">
+        <strong editable-text="meta.label" onbeforesave="ic.saveMetadata(meta.label, meta.value)">{{ meta.label || "No Value" }}</strong>:
+        <span editable-text="meta.value" onbeforesave="ic.saveMetadata(meta.label, meta.value)">{{ meta.value || "No Value" }}</span>
```

Comment:
Shouldn't we be passing `$index` to the `saveMetadata()`? Since the metadata is stored as an array, the index would identify the label/value pair.